README.md

```diff
@@ -1,6 +1,8 @@
 # Accessibility Statement Frontend
 
-> **Warning**
+> [!CAUTION]
```

Comment:
this is the new admonition format https://github.com/orgs/community/discussions/16925